README.md

```diff
@@ -536,6 +536,154 @@ $mailersend = new MailerSend(['api_key' => 'key']);
 $mailersend->recipients->delete('recipient_id');
 ```
 
+**Add recipients to a suppression list**
+
+**Blocklist**
+
+```php
+use MailerSend\MailerSend;
+use MailerSend\Helpers\Builder\BlocklistParams;
+
+$mailersend = new MailerSend(['api_key' => 'key']);
+
+$params = (new BlocklistParams())
+    ->setDomainId('domain_id')
+    ->setRecipients(['recipient_one', 'recipient_two'])
+    ->setPatterns(['pattern_one', 'pattern_two']);
+
+$mailersend->blocklist->create($params);
+```
+
+**Hard Bounces**
+
+```php
+use MailerSend\MailerSend;
+
+$mailersend = new MailerSend(['api_key' => 'key']);
+
+$mailersend->hardBounce->create('domain_id', ['recipient_one', 'recipient_two']);
+```
+
+**Spam Complaints**
+
+```php
+use MailerSend\MailerSend;
+
+$mailersend = new MailerSend(['api_key' => 'key']);
+
+$mailersend->spamComplaint->create('domain_id', ['recipient_one', 'recipient_two']);
+```
+
+**Unsubscribes**
+
+```php
+use MailerSend\MailerSend;
+
+$mailersend = new MailerSend(['api_key' => 'key']);
+
+$mailersend->unsubscribe->create('domain_id', ['recipient_one', 'recipient_two']);
+```
```

Comment:
This seems wrong, these should be using the same Blocklist params, no?

Reply:
Yes! I did the change and forgot to update that 🤦